### ISSUE TITLE
stationxml: DOI extension

### ISSUE THREAD

**Issue by megies** [obspy/obspy#856]
It seems that the extension for DOIs is also finalized now (http://www.iris.washington.edu/pipermail/fdsn-wg3-products/2014-July/000063.html) although there is no note on the FDSN StationXML page yet.. if it indeed is finalized already it would be good if this gets in 0.10.0.

**Comment by petrrr** [comment]
Hi all, 
there is some subtle detail I would like to clarify with the authors of the document "FDSN recommendations for seismic network DOIs and related FDSN services", which regards the example only. While the scheme and the rest of the document should be okay. 

I believe that the "identifier" in the element should appear "plain", that is without the print prefix "doi:", which strictly is not part of the identifier itself, but is the way DOIs are supposed to appear in printed or online (human-readable) documents. However, (1) in the context of StationXML the exact use of these PIDs could be anything, not necessary written out representation, but could be resolution, machine readable reference etc. (2) Other PIDs / PID systems might not define an equivalent prefix scheme, which would make the use inconsistent, (3) it is redundant with the type attribute.

So I believe the example should read:

```
<Network code="CO">
  <Identifier type="DOI">10.1341/abcde</Identifier>
  <Identifier type="ARK">12341515</Identifier>
  <Identifier type="EPIC">110099/00-2fc01799d1f</Identifier>
  ...
</Network>
```

**Comment by megies** [comment]
> I would like to clarify with the authors of the document "FDSN recommendations for seismic network DOIs and related FDSN services"



> I believe that the "identifier" in the element should appear "plain", that is without the print prefix "doi:"

I agree, but you should probably bring your point forward here: https://github.com/FDSN/StationXML

**Comment by petrrr** [comment]
@megies: I brought this point to the attention of one of the authors (Angelo), asking to clarify on this. 

I am not 100% sure if the repo you indicated is really the right place for the discussion. I may be wrong, but this is about the validation of the XML, which to my understanding is okay.

**Comment by megies** [comment]
Well, it's either there at github or via the fdsn mailing lists (WG3? WG2?).

**Comment by megies** [comment]
See FDSN/StationXML#3.

**Comment by petrrr** [comment]
I think the change was officially approved, see also http://www.fdsn.org/message-center/thread/399/.

**Comment by megies** [comment]
> I think the change was officially approved, see also http://www.fdsn.org/message-center/thread/399/.

Yes, but it's not in a released version yet. Also we'll have to decide how to deal with different versions when adding this..

**Comment by megies** [comment]
It seems this is not yet in a StationXML release version, so I'll bump it to 1.2.0.

**Comment by megies** [comment]
Pretty sure this is already fixed on 1.2.0 and above
